mobile-widgets/qmlmanager.cpp

```diff
@@ -598,6 +608,9 @@ QMLManager::~QMLManager()
 #if defined(Q_OS_ANDROID) || defined(Q_OS_IOS)
 	if (appLogFileOpen)
 		appLogFile.close();
+#endif
+#if defined (Q_OS_IOS)
+	delete iosshare;
```

Comment:
Not needed if using a subobject or a unique_ptr.

Reply:
nor when just using a sub-object